coding-concepts.js

```diff
@@ -9,46 +9,46 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 5
+// b) Verify and explain: .push() adds the element to the end and returns the  new value of the array
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 9 
+// b) Verify and explain: It will log the length of the string, starting from 0 index.
```

Comment:
`.length` will return the length of the string, which will count the amount of characters in the string. Remember that the space also counts as a character! So this will return 10.